stories/Section/Section.stories.js

```diff
@@ -0,0 +1,255 @@
+            ipsum faucibus.
+          </Col>
+          <Col xs={'12'} lg={'6'} xl={'4'} className="pr-0 pr-md-5 mb-3">
+            Euismod lacinia at quis risus sed vulputate. Scelerisque purus
+            semper eget duis at tellus at urna condimentum. Mattis enim ut
+            tellus elementum sagittis.
+          </Col>
+        </Row>
+      </Container>
+    </Section>
+  )
+}
+
+const SfondoPrimarioComponent = () => {
+  return (
+    <Section color="primary">
+      <Container className="white-color">
+        <Row className="mb-3">
+          <Col xs={'12'}>
+            <h4>Morbi fermentum amet</h4>
+          </Col>
+        </Row>
+        <Row>
+          <Col xs={'12'} lg={'6'} xl={'4'} className="pr-0 pr-md-5 mb-3">
+            Platea dictumst vestibulum rhoncus est pellentesque elit ullamcorper
+            dignissim cras. Dictum sit amet justo donec enim diam vulputate ut.
+            Eu nisl nunc mi ipsum faucibus.
+          </Col>
+          <Col xs={'12'} lg={'6'} xl={'4'} className="pr-0 pr-md-5 mb-3">
+            Eget egestas purus viverra accumsan. Diam maecenas ultricies mi eget
+            mauris pharetra et. Etiam dignissim diam quis enim. Eu nisl nunc mi
+            ipsum faucibus.
+          </Col>
+          <Col xs={'12'} lg={'6'} xl={'4'} className="pr-0 pr-md-5 mb-3">
+            Euismod lacinia at quis risus sed vulputate. Scelerisque purus
+            semper eget duis at tellus at urna condimentum. Mattis enim ut
+            tellus elementum sagittis.
+          </Col>
+        </Row>
+      </Container>
+    </Section>
+  )
+}
+
+const SfondoNeutraleComponent = () => {
+  return (
+    <Section neutral>
+      <Container className="white-color">
+        <Row className="mb-3">
+          <Col xs={'12'}>
+            <h4>Morbi fermentum amet</h4>
+          </Col>
+        </Row>
+        <Row>
+          <Col xs={'12'} lg={'6'} xl={'4'} className="pr-0 pr-md-5 mb-3">
+            Platea dictumst vestibulum rhoncus est pellentesque elit ullamcorper
+            dignissim cras. Dictum sit amet justo donec enim diam vulputate ut.
+            Eu nisl nunc mi ipsum faucibus.
+          </Col>
+          <Col xs={'12'} lg={'6'} xl={'4'} className="pr-0 pr-md-5 mb-3">
+            Eget egestas purus viverra accumsan. Diam maecenas ultricies mi eget
+            mauris pharetra et. Etiam dignissim diam quis enim. Eu nisl nunc mi
+            ipsum faucibus.
+          </Col>
+          <Col xs={'12'} lg={'6'} xl={'4'} className="pr-0 pr-md-5 mb-3">
+            Euismod lacinia at quis risus sed vulputate. Scelerisque purus
+            semper eget duis at tellus at urna condimentum. Mattis enim ut
+            tellus elementum sagittis.
+          </Col>
+        </Row>
+      </Container>
+    </Section>
+  )
+}
+
+const ImmagineComponent = () => {
+  return (
+    <Section image backgroundImage="https://picsum.photos/1280/720?image=811">
```

Comment:
Just one note here: `image` seems to not be a correct prop here.
I would rename `backgroundImage` to `image` with the same semantic.

Reply:
Done!